src/main/java/seedu/duke/exception/TooManyVariablesException.java

```diff
@@ -0,0 +1,4 @@
+package seedu.duke.exception;
+
+public class TooManyVariablesException extends Exception{
```

Comment:
Perhaps put a space in between the closing parenthesis and the opening curly braces